ci/build.yml

```diff
@@ -275,10 +275,31 @@ jobs:
           echo "Could not connect to Oracle, trying again..."
           sleep 1
         done
+
+
+        # Setting MAX_STRING_SIZE='EXTENDED' in order to enable specfying collation at column level.
+        # See 'Increasing the Maximum Size of VARCHAR2, NVARCHAR2, and RAW Columns in a PDB' at
+        # https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/MAX_STRING_SIZE.html#GUID-D424D23B-0933-425F-BC69-9C0E6724693C
+        #
+        cat <<EOT > set_max_string_size.txt
+         SHOW PARAMETER max_string_size;
+         SHUTDOWN IMMEDIATE;
+         ALTER PLUGGABLE DATABASE orclpdb1 OPEN UPGRADE;
+         ALTER SYSTEM SET max_string_size='EXTENDED';
+         @?/rdbms/admin/utl32k.sql;
+         @?/rdbms/admin/utlrp.sql;
+         SHUTDOWN IMMEDIATE;
+         STARTUP;
+         SHOW PARAMETER max_string_size;
+         ;
+        EOT
+        docker cp set_max_string_size.txt oracle:/home/oracle/set_max_string_size.txt
+        docker exec oracle bash -c 'sqlplus -L sys/'"$ORACLE_PWD"'@//localhost:'"$ORACLE_PORT"'/ORCLPDB1 as sysdba <<< "@set_max_string_size.txt" '
+
         # Actually run some tests
         # Note: Oracle tests all run sequentially because they all access the same Oracle instance,
         # and we sometimes observe transient connection issues when running tests in parallel.
-        bazel test \
+        bazel test -t- \
```

Comment:
What is this option doing?

Reply:
Temporary mean to prevent Bazel from caching and thus not executing tests on Oracle.
I'm guessing it's similar or equivalent `--cache_test_results=no`